service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -0,0 +1,455 @@
+
+.Authentication
+
+Additional authentication methods specified in `spec.origins`, must be mapped into a `security.istio.io/v1beta1` RequestAuthentication resource.  `spec.selector.matchLabels` should be configured similarly to the same field on PeerAuthentication. Configuration specific to JWT principals from `spec.origins.jwt` items map to similar fields in spec.rules items.
+
+* `spec.origins[x].jwt.triggerRules` specified in the Policy will need to be mapped into one or more `security.istio.io/v1beta1` AuthorizationPolicy resources. Any `spec.selector.labels` should be configured similarly to the same field on RequestAuthentication.  * spec.origins[x].jwt.triggerRules.excludedPaths should be mapped into an AuthorizationPolicy whose spec.action is set to ALLOW, with `spec.rules[x].to.operation.path` entries matching the excluded paths.  `spec.origins[x].jwt.triggerRules.includedPaths` should be mapped into a separate AuthorizationPolicy whose `spec.action` is set to `ALLOW`, with `spec.rules[x].to.operation.path` entries matching the included paths, and `spec.rules.[x].from.source.requestPrincipals` entries that align with the `specified spec.origins[x].jwt.issuer` in the Policy resource.
+
+.ServiceMeshPolicy (maistra.io/v1)
+
+ServiceMeshPolicy was configured automatically for the control plane through the `spec.istio.global.mtls.enabled` in the v1 resource or `spec.security.dataPlane.mtls` in the v2 resource setting. For v2 control planes, a functionally equivalent PeerAuthentication resource will be created during installation.
+
+.RbacConfig, ServiceRole, ServiceRoleBinding (rbac.istio.io/v1alpha1)
+
+These resources have been replaced by the `security.istio.io/v1beta1` AuthorizationPolicy resource.
+
+Mimicking RbacConfig behavior requires writing a default AuthorizationPolicy whose settings depend on the spec.mode specified in the RbacConfig.
+
+* When `spec.mode` is set to `OFF`, no resource is required as the default policy is ALLOW, unless an AuthorizationPolicy applies to the request.
+* When `spec.mode` is set to ON, set `spec: {}`.  You must create AuthorizationPolicy policies for all services in the mesh.
+* `spec.mode` is set to `ON_WITH_INCLUSION`, must create an AuthorizationPolicy with `spec: {}` in each included namespace. Inclusion of individual services is not supported by AuthorizationPolicy. However, as soon as any AuthorizationPolicy is created that applies to the workloads for the service, all other requests not explicitly allowed will be denied.
+* `spec.mode` is set to `ON_WITH_EXCLUSION`, is not supported by AuthorizationPolicy. A global DENY policy can be created, but an AuthorizationPolicy must be created for every workload in the mesh because there is no allow-all policy that can be applied to either a namespace or a workload.
```

Comment:
This and the previous sound odd to me: 

Should it be: 

Setting `spec.mode` to `ON_WITH_EXCLUSION`, is not supported by AuthorizationPolicy.

or something similar? The placement of the two instances of `is` doesn't sound correct.